### PR TITLE
[INLONG-2109][manager] support db SQL collect

### LINKED ISSUE #2109: [Feature] [inlong manager] support db collect
### Description

[inlong manager] support db collect. 
1. support task dispatch
2. support task state report

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by justinwwhuang**
1.support task dispatch
2.support task state report

### Title Name: [INLONG-XYZ][component] Title of the pull request

where *XYZ* should be replaced by the actual issue number.

Fixes #2109

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Review by gosonzhang — APPROVED** [review]

**Review by baomingyu — APPROVED** [review]

**Review by EMsnap — CHANGES_REQUESTED** [review]
pls check my review

File: inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/fetcher/dtos/SqlJobProfileDto.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        job.setId(Integer.parseInt(task.getId()));
+        job.setName(SQL_JOB);
+        job.setSource(DEFAULT_SOURCE);
+        job.setSink(DEFAULT_BUS_SINK);
+        job.setChannel(DEFAULT_CHANNEL);
+        //job.setTimeOffset(dataConfigs.getTimeOffset());
+        //job.setOp(dataConfigs.getOp());
+        //job.setDeliveryTime(dataConfigs.getDeliveryTime());
+        //if (!dataConfigs.getAdditionalAttr().isEmpty()) {
+        //    job.setAddictiveString(dataConfigs.getAdditionalAttr());
```
[the remark this answers is not in the document]
Comment (justinwwhuang) [reply inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/fetcher/dtos/SqlJobProfileDto.java:105]: ok

File: inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/fetcher/dtos/DbCollectorTask.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+@Data
+public class DbCollectorTask {
+
+    private String id;
+    private Integer type;
+    private Integer dbType;
+    private String ip;
+    private Integer port;
+    private String dbName;
+    private String user;
```
[the remark this answers is not in the document]
Comment (justinwwhuang) [reply inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/fetcher/dtos/DbCollectorTask.java:31]: ok，i will add a field named seperator at the end

File: inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/fetcher/dtos/SqlJobProfileDto.java [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+        //private Thread thread;
+        private String name;
+        private String source;
+        private String sink;
+        private String channel;
+        //private String pattern;
+        //private String op;
+        //private String cycleUnit;
+        //private String timeOffset;
+        //private String deliveryTime;
```
[the remark this answers is not in the document]
Comment (justinwwhuang) [reply inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/fetcher/dtos/SqlJobProfileDto.java:65]: ok

File: inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/MiniAgent.java [hunk 4]
```diff
@@ -46,7 +46,7 @@ public void start() throws Exception {
     }
 
     public void submitJob(JobProfile profile) {
-        manager.getJobManager().submitJobProfile(profile);
+        manager.getJobManager().submitFileJobProfile(profile);
```
[the remark this answers is not in the document]
Comment (justinwwhuang) [reply inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/MiniAgent.java:49]: there are many places using job or jobprofile，we will see how to unite

**Review by EMsnap — APPROVED** [review]
+1

**Review by dockerzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]
+1

**Pull request merged**